migrate.py

```diff
@@ -1198,6 +1146,57 @@ def assemble_collections(checkout_path, spec, args, target_github_org):
             REMOVE = set()
 
 
+def galaxy_metadata_init(collection, namespace, target_github_org):
+    return {
+        'namespace': namespace,
+        'name': collection,
+        'version': '1.0.0',  # TODO: add to spec, args?
+        'readme': None,
+        'authors': None,
+        'description': None,
+        'license': None,
+        'license_file': None,
+        'tags': None,
+        'dependencies': {},
+        'repository': f'git@github.com:{target_github_org}/{namespace}.{collection}.git',
+        'documentation': None,
+        'homepage': None,
+        'issues': None
+    }
+
+
+def process_symlink(namespace, collection, plugin_type, dest, src, spec):
+    real_src = os.readlink(src)
+
+    # remove destination if it already exists
+    if os.path.exists(dest):
+        # NOTE: not atomic but should not matter in our script
+        logger.warning('Removed "%s" as it is target for symlink of "%s"' % (dest, src))
+        os.remove(dest)
+
+    if real_src.startswith('../'):
+        target = real_src[3:]
+        found = False
+        for k in spec[namespace][collection][plugin_type]:
+            if k.endswith(target):
+                found = True
+                break
+
+        if found:
```

Comment:
Let's invert this as:
```suggestion
        if not found:
            raise LookupError(
                'Found symlink "%s" to target "%s" that is not in the same collection.',
                src, target,
            )
```
which would allow us to dedent the whole else-block.